lib/readline.js

```diff
@@ -907,7 +908,7 @@ exports.Interface = Interface;
 const KEYPRESS_DECODER = Symbol('keypress-decoder');
 const ESCAPE_DECODER = Symbol('escape-decoder');
 
-function emitKeypressEvents(stream) {
+function emitKeypressEvents(stream, iface) {
```

Comment:
this is now public and requires documentation https://nodejs.org/dist/latest-v6.x/docs/api/readline.html#readline_readline_emitkeypressevents_stream